package.json

```diff
@@ -248,5 +252,24 @@
   "lint-staged": {
     "*.{css,scss}": "stylelint --fix",
     "*.{js,jsx,ts,tsx,css,scss,md,html}": "prettier --write"
+  },
+  "packageManager": "pnpm@8.8.0+sha256.d713a5750e41c3660d1e090608c7f607ad00d1dd5ba9b6552b5f390bf37924e9",
+  "pnpm": {
+    "peerDependencyRules": {
+      "allowedVersions": {
+        "@typescript-eslint/parser": "^6.2.0"
+      }
+    },
+    "allowedDeprecatedVersions": {
+      "@fortawesome/fontawesome-svg-core": "*"
+    },
+    "updateConfig": {
+      "ignoreDependencies": [
+        "fork-ts-checker-notifier-webpack-plugin",
+        "fork-ts-checker-webpack-plugin",
+        "@fortawesome/fontawesome-free",
+        "@fortawesome/fontawesome-svg-core"
+      ]
+    }
```

Comment:
This stuff is pretty neat - you can remove things from the output of `npm outdated`, silence deprecation warnings, and correct outdated peer dependencies, among others.